src/main/java/ti4/commands/bothelper/CreateFOWGameChannels.java

```diff
@@ -40,7 +40,7 @@ public CreateFOWGameChannels() {
         addOptions(new OptionData(OptionType.USER, Constants.PLAYER6, "Player6 @playerName"));
         addOptions(new OptionData(OptionType.USER, Constants.PLAYER7, "Player7 @playerName"));
         addOptions(new OptionData(OptionType.USER, Constants.PLAYER8, "Player8 @playerName"));
-        addOptions(new OptionData(OptionType.USER, Constants.FOWGM, "Default GM is whoever runs this command"));
+        addOptions(new OptionData(OptionType.USER, Constants.FOWGM, "Game master (default: you)"));
```

Comment:
GM is a more recognizable term than Game Master. My brain doesnt even recognize thats what GM stands for, and those two words do not get used in the fog world, whereas GM gets used a lot.

Reply:
Changed to "GM/game master".